pygfx/pkg_resources/__init__.py

```diff
@@ -0,0 +1 @@
+# @almarklein: why do we need this init file here?
```

Comment:
It relates to the packaging setup. This folder needs to be recognized as a python module. We load the binaries from this folder via package_resources, and it resolves paths by module names.

Reply:
Ah right. There is some logic in `pygfx.utils._dirs` that relies on the folder being a module to access resources. Thanks for clarifying